### PR TITLE
finished work

### PR DESCRIPTION
**Pull request by stottidfe**
[NSA](https://dfe-secureaccess.atlassian.net/browse/NSA-7109)
made changes to the sub-service approved email both html and text

### CODE REVIEW

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs [hunk 1]
```diff
@@ -193,9 +193,9 @@
                 <p>Dear <%=firstName%> <%=lastName%>,</p>
 
                 <% if (isMultipleSubServicesReq) { %>
-                    <p>New sub-services have been added to your DfE Sign-in account.</p>
+                    <p>Your request to change sub-services access has been approved.</p>
```
Comment (ipredoi) [inline lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs:196]: In the prototype we have only `Your request to change sub-service access has been approved.` - if already checked with the UI/UX designer, please ignore this comment

Comment (stottidfe) [reply lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs:196]: checked with matt when we had the some sub-services only one item and some have multiples and noticed that in certain parts of the html and text it looks for multiples he said if we can then we should

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs [hunk 2]
```diff
@@ -255,17 +255,17 @@
                     </div>
                 </dl>
           
-
-                <p>To access this sub-service, sign in to your DfE Sign-in account at <a href="<%=signInUrl%>">services.sign-in</a>. Or, you can select the button below to sign in to your account.</p>
-
+                <% if (isMultipleSubServicesReq) { %>
+                    <p>To access your sub-services, sign in to your DfE Sign-in account at <a href="<%=signInUrl%>">services.sign-in</a>. Or, you can select the button below to sign in to your account.</p>
```
Comment (ipredoi) [inline lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs:259]: as per the prototype, `sign in to` should be `sign into` - please check with the content designer

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs [hunk 3]
```diff
@@ -255,17 +255,17 @@
...
+                <% } else { %>
+                    <p>To access your sub-service, sign in to your DfE Sign-in account at <a href="<%=signInUrl%>">services.sign-in</a>. Or, you can select the button below to sign in to your account.</p>
```
Comment (ipredoi) [inline lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs:261]: as per the prototype, sign in to should be sign into - please check with the content designer

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs (hunk 2 above)
Comment (stottidfe) [reply lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs:259]: fixed with next checkin

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs (hunk 3 above)
Comment (stottidfe) [reply lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs:261]: fixed with next checkin

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs [hunk 4]
```diff
@@ -255,17 +255,17 @@
...
+                <% } %>
                 <p>
-                <a class="button-link" href="<%=signInUrl%>">Go to your DfE Sign-in account</a>
+                <a class="button-link" href="<%=signInUrl%>">Go to DfE Sign-in account</a>
```
Comment (ipredoi) [inline lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs:264]: as per the prototype, this should be `Go to DfE Sign-in` and not `Go to DfE Sign-in account`

Comment (stottidfe) [reply lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs:264]: fixed in next checkin

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs [hunk 5]
```diff
@@ -255,17 +255,17 @@
...
                 </p>
-                <p>Once signed in, select the service this sub-service applies to from the list on your services page.</p>
                 <p>
                     Kind regards,
```
Comment (ipredoi) [inline lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs:267]: As per the prototype, there is no comma after Kind regards, please double check with the UI/UX  designer 
image

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/text.ejs
Comment (ipredoi) [inline lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/text.ejs:1]: same content remarks as lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs (hunk 5 above)
Comment (stottidfe) [reply lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs:267]: removed as per design fixed next checkin

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/text.ejs
Comment (stottidfe) [reply lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/text.ejs:1]: fixed next checkin

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs (hunk 1 above)
Comment (stottidfe) [reply lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs:196]: this is done

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs [hunk 6]
```diff
@@ -255,17 +255,17 @@
                     </div>
                 </dl>
           
-
-                <p>To access this sub-service, sign in to your DfE Sign-in account at <a href="<%=signInUrl%>">services.sign-in</a>. Or, you can select the button below to sign in to your account.</p>
-
+                <% if (isMultipleSubServicesReq) { %>
+                    <p>To access your sub-services, sign into your DfE Sign-in account at <a href="<%=signInUrl%>">services.sign-in</a>. Or, you can select the button below to sign in to your account.</p>
```
Comment (ipredoi) [inline lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs:259]: please modify the content as per the protype 
image

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs [hunk 7]
```diff
@@ -255,17 +255,17 @@
...
+                <% } else { %>
+                    <p>To access your sub-service, sign into your DfE Sign-in account at <a href="<%=signInUrl%>">services.sign-in</a>. Or, you can select the button below to sign in to your account.</p>
```
Comment (ipredoi) [inline lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs:261]: please modify the content as per the protoype
image

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/text.ejs [hunk 8]
```diff
@@ -21,16 +21,17 @@ Sub-service: <% if (requestedSubServices.length > 0) { %>
               No roles selected
             <% } %>
 Status: Approved
+<% if (isMultipleSubServicesReq) { %>
+To access this sub-services, sign into your DfE Sign-in account at <%=signInUrl%>. Or, you can select the button below to sign in to your account.
```
Comment (ipredoi) [inline lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/text.ejs:25]: please update the content to match the prototype 
image

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/text.ejs [hunk 9]
```diff
@@ -21,16 +21,17 @@ Sub-service: <% if (requestedSubServices.length > 0) { %>
...
+<% } else { %>
+To access this sub-service, sign into your DfE Sign-in account at <%=signInUrl%>. Or, you can select the button below to sign in to your account.
```
Comment (ipredoi) [inline lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/text.ejs:27]: image

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/text.ejs (hunk 8 above)
Comment (stottidfe) [reply lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/text.ejs:25]: done

File: lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs (hunk 6 above)
Comment (stottidfe) [reply lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs:259]: done

**Review by ipredoi — APPROVED** [review]

**Pull request merged**